src/fido/types.h

```diff
@@ -157,6 +158,7 @@ typedef struct _fido_assert_stmt {
 	fido_authdata_t authdata;        /* decoded authdata payload */
 	fido_blob_t     sig;             /* signature of cdh + authdata */
 	fido_blob_t     largeblob_key;   /* decoded large blob key */
+	fido_blob_t     blob;            /* decoded cred blob */
```

Comment:
s/decoded cred blob/decoded FIDO 2.1 credBlob/, maybe?

Reply:
fixed in 105034ed49b1f6c2c2b8c3e32b770366a911625f